src/Symfony/Component/Validator/Constraints/AtLeastOneOfValidator.php

```diff
@@ -34,6 +34,10 @@ public function validate($value, Constraint $constraint)
         $messages = [$constraint->message];
 
         foreach ($constraint->constraints as $key => $item) {
+            if (!in_array($this->context->getGroup(), $item->groups, true)) {
```

Comment:
Is there is similar issue somehow with the [`SequentiallyValidator`](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Validator/Constraints/SequentiallyValidator.php), or is this only necessary because of the [clone](https://github.com/symfony/symfony/pull/39872/files#diff-ffba2bed571e18a91782556cc0ca76152bcb556bd1a83590e733f4784e7c88f4R368-R371) method which does only copy violation now?

Reply:
The code works as expected if I don't miss anything (I am going to investigate what's the actual difference). But I took the chance and added another test case to the `SequentiallyValidatorTest` class to make sure that we not break it there in the future.

Reply:
Got it, the explanation is that the exit condition is different. In `AtLeastOneOfValidator` we abort as soon as we have a constraint that does not add any additional violation (which is always the case if the constraint is not part of the current group).

Reply:
Alright, thx for the double-check